Mathlib/Algebra/Homology/ExactSequence.lean

```diff
@@ -308,6 +308,14 @@ lemma exact_of_δlast {n : ℕ} (S : ComposableArrows C (n + 2))
   rw [exact_iff_δlast]
   constructor <;> assumption
 
+lemma Exact.isIso_map' {C : Type*} [Category C] [Preadditive C]
+    [Balanced C] {n : ℕ} {S : ComposableArrows C n} (hS : S.Exact) (k : ℕ) (hk : k + 3 ≤ n)
```

Comment:
```suggestion
    [Balanced C] {n : ℕ} {S : ComposableArrows C n} (hS : S.Exact) {k : ℕ} (hk : k + 3 ≤ n)
```
Feel free to ignore if in the applications it is better to have it explicit.

Reply:
Here, I think it should be explicit. (There might by away to the `hk` field automatically filled by `omega`, but I am not sure how to do that as `hk` is used in auto-filled arguments in `h₀` and `h₁`.)